### PR TITLE
Upgrade govuk frontend to v5

### PR DESCRIPTION
**Pull request by kr8n3r**
## What 

Update the app to use govuk-frontend v5.4.0 for WCAG 2.2 compliance.
See individual commits for detailed changes.

Addresses https://trello.com/c/aTQLmIRB/856-bump-design-system-to-v5-in-document-download

Screenshot 2024-07-11 at 12 35 31
Screenshot 2024-07-11 at 12 34 57
Screenshot 2024-07-11 at 12 35 18

---

🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨

For more information, including how to check this deployment on preview or staging first before it goes to production, see our [team wiki section on deployment](https://github.com/alphagov/notifications-manuals/wiki/Merging-and-deploying#deployment)

### CODE REVIEW

File: app/assets/javascripts/main.mjs [hunk 1]
```diff
@@ -1,7 +1,15 @@
-import { Frontend } from './govuk-frontend-local';
+import { Button, createAll, ErrorSummary, SkipLink } from 'govuk-frontend'
 
-Frontend.initAll();
+createAll(Button)
 
-export {
-  Frontend
+// Find first error summary module to enhance.
```
Comment (tombye) [inline app/assets/javascripts/main.mjs:5]: Should we just use `createAll` to instatiate this module and the `$skipLink`? The guidance seems to say we should only not use it if we intend to handle any errors thrown ourselves, which we don't.

**Review by tombye — CHANGES_REQUESTED** [review]
I added a comment and we should also stop using the 'element' param with the `govukButton` component because it was deprecated in V5.1.0: https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0

File: app/assets/javascripts/main.mjs (hunk 1 above)
Comment (kr8n3r) [reply app/assets/javascripts/main.mjs]: yep, correct we can just use that. updated

**Comment by kr8n3r** [comment]
> I added a comment and we should also stop using the 'element' param with the `govukButton` component because it was deprecated in V5.1.0: https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0

create spot! removed from the button

**Review by tombye — APPROVED** [review]
Thanks for doing the fixes. Great, speedy work 👍🏻

**Pull request merged**